packages/neuron-ui/src/locales/en.ts

```diff
@@ -33,9 +46,7 @@ export default {
       detail: 'detail',
       explorer: 'explorer',
     },
-    transaction: {
-      goBack: 'go back',
-    },
+    transaction: { goBack: 'go back' },
```

Comment:
Why not `go-back` 🤔

Reply:
previously we use camelCase, several days ago we are going to use hyphen-seperated-case. We'd better to issue a new task to format i18n keys.